EESSI-install-software.sh

```diff
@@ -187,6 +187,22 @@ fi
 # assume there's only one diff file that corresponds to the PR patch file
 pr_diff=$(ls [0-9]*.diff | head -1)
 
+# install any additional required scripts
+# order is important: these are needed to install a full CUDA SDK in host_injections
+install_scripts_changed=$(cat ${pr_diff} | grep '^+++' | cut -f2 -d' ' | sed 's@^[a-z]/@@g' | grep '^install_scripts.sh$' > /dev/null; echo $?)
```

Comment:
@casparvl I messed up a little by merging this as now the file checked here will not be updated in the other PR, and as a result the scripts will not get installed. It doesn't matter too much as I think the use of `$pr_diff` is overkill here. I think we should always run the installer script and instead of using `cp` use `cp -u`, that could be implemented as part of #434

Also, this was a little wrong anyway, as it doesn't check if the scripts _installed_ by the installer have changed